cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -1236,14 +1235,14 @@ def _load_multiplexed_pdu(self, pdu, frame_name, next_selector_idx):
             # specified indepently of that of the message. how should
             # this be handled?
 
-        if selector_signal.initial in selector_signal.choices:
-            selector_signal.initial = \
-                selector_signal.choices[selector_signal.initial]
+        if selector_signal.raw_initial in selector_signal.choices:
+            selector_signal.raw_initial = \
+                selector_signal.choices[selector_signal.raw_initial]
 
-        if not isinstance(selector_signal.invalid, NamedSignalValue) and \
-           selector_signal.invalid in selector_signal.choices:
-            selector_signal.invalid = \
-                selector_signal.choices[selector_signal.invalid]
+        if not isinstance(selector_signal.raw_invalid, NamedSignalValue) and \
```

Comment:
In my understanding, `raw_` values should never be `NamedSignalValue`s? (rather something like `Union[int, float, bytes]`?)